Cargo.toml

```diff
@@ -116,6 +116,8 @@ which = "5.0.0"
 strip-ansi-escapes = "0.2.0"
 proptest = "1.5.0"
 hex-literal = "0.4.1"
+sc-executor = "0.40.0"
+sc-executor-common = "0.35.0"
```

Comment:
Very much a Nit, but these are substrate crates too so could be below with the rest :)